### PR TITLE
add property

### PR DESCRIPTION
**Pull request by LucasColas**
Hello,
I added a property for the attribute called action. I think it's useful.
Because sometimes you can need to have access to the last action. self._action is supposed to be encapsulated.

### CODE REVIEW

File: texasholdem/game/game.py [hunk 1]
```diff
@@ -291,6 +291,10 @@ def __init__(self, buyin: int, big_blind: int, small_blind: int, max_players=9):
         self.hand_history: Optional[History] = None
         self._action = None, None
         self._hand_gen = None
+        
+    @property        
+    def action(self) -> tuple:
```
Comment (SirRender00) [inline texasholdem/game/game.py:296]:
```suggestion
    def action(self) -> Tuple[ActionType, Optional[int]]:

```

**Review by SirRender00 — CHANGES_REQUESTED** [review]
Sounds reasonable 👍. You'll need to add a docstring on the function AND at the top of the class, you can add it to the `attributes` for the docstring of the class.

**Comment by LucasColas** [comment]
> Sounds reasonable 👍. You'll need to add a docstring on the function AND at the top of the class, you can add it to the `attributes` for the docstring of the class.

Done.

File: texasholdem/game/game.py [hunk 2]
```diff
@@ -291,6 +292,14 @@ def __init__(self, buyin: int, big_blind: int, small_blind: int, max_players=9):
         self.hand_history: Optional[History] = None
         self._action = None, None
         self._hand_gen = None
+
```
Comment (SirRender00) [inline texasholdem/game/game.py:295]:
```suggestion


```

**Comment by SirRender00** [comment]
@LucasColas Also feel free to run pylint locally with `poetry run pylint texasholdem/`

**Comment by SirRender00** [comment]
@LucasColas Looks like you have to run `poetry run black .` to format the code.

**Comment by LucasColas** [comment]
I'll do it.

**Comment by LucasColas** [comment]
To be honest, I don't really understand how to install poetry.

**Comment by LucasColas** [comment]
Ok, I think I understood. I did a new commit.

**Comment by LucasColas** [comment]
How to handle codecov ?

**Review by SirRender00 — APPROVED** [review]

**Comment by SirRender00** [comment]
Ah yeah don't worry about codecov, I'll merge this in 👍

**Pull request merged**